packages/app/obojobo-repository/shared/components/notification.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react'
+import { useState, useEffect } from 'react'
```

Comment:
This could be shortened as `const React, { useEffect, useState} = require('react')`.

You could also just `const React = require('react')` and use `React.useState` and `React.useEffect` in the code instead, if you're only going to do so once or twice.